ovs/lib/vdisk.py

```diff
@@ -62,6 +62,8 @@
 storagerouterclient.Logger.setupLogging(LogHandler.load_path('storagerouterclient'))
 storagerouterclient.Logger.enableLogging()
 
+METADATA_CACHE_PAGE_SIZE = 256 * 24
```

Comment:
I'd add these to the ```StorageDriverClient```, as it currently already contains a lot of stuff like defaults etc.